BUILD.md

```diff
@@ -426,7 +435,7 @@ The DirectML execution provider supports building for both x64 and x86 architect
 ### ARM Compute Library
 See more information on the ACL Execution Provider [here](./docs/execution_providers/ACL-ExecutionProvider.md).
 
-#### Prerequisites
+#### Pre-Requisites
```

Comment:
>Pre-Requisites [](start = 5, length = 14)

either don't put a hyphen but if you are putting then 'r' needs to small not capital.